style.css

```diff
@@ -89,3 +139,705 @@ body {
+}
+
+button:enabled {
+  background: #f55800;
+  color: #fff;
+}
+
+button:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+  cursor: pointer;
+}
+
+button:active {
+  background: #e05304;
+  color: #f4f1f0;
+}
+
+button:disabled {
+  background: #f4f1f0;
+  color: #5b5957;
+}
+
+.projectBody {
+  width: 295px;
+  height: 87px;
+  margin: 80px 16px 12px 16px;
+}
+
+.projectBody p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.projectBody ul {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  margin-top: 5px;
+  margin-bottom: 22px;
+}
+
+.projectBody ul li {
+  list-style: none;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  margin-right: 12px;
+  position: static;
+}
+
+.projectHTML {
+  width: 56px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+  margin-right: 24px;
+}
+
+.projectHTML ul li {
+  width: 32px;
+  height: 20px;
+  padding: 10px 12px;
+  color: #fff;
+  position: relative;
+  right: 39px;
+}
+
+.projectBootstrap {
+  width: 95px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+  margin-left: -24px;
+  margin-right: 12px;
+}
+
+.projectBootstrap ul li {
+  width: 71px;
+  height: 20px;
+  padding: 10px 12px;
+  color: #fff;
+  margin-right: 30px;
+  position: relative;
+  right: 39px;
+}
+
+.projectRuby {
+  width: 61px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+  margin-right: 12px;
+  margin-left: -10px;
+}
+
+.projectRuby ul li {
+  width: 37px;
+  height: 20px;
+  padding: 10px 12px;
+  color: #fff;
+  position: relative;
+  right: 39px;
+}
+
+.technologies {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  text-align: center;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  padding-inline-start: 0;
+}
+
+.technologies ul li {
+  text-align: left;
+}
+
+.technologies li > ul > li {
+  margin-bottom: 32px;
+  height: 24px;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  color: #fff;
+}
+
+.technologies h4 {
+  font-family: 'InaiMathi', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  display: block;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  padding: 44px 0 44px 0;
+}
+
+#list-menu li {
+  border-bottom: 1px solid #6f6c6b;
+}
+
+#list-menu a {
+  text-decoration: none;
+  color: #fff5e1;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+#delete {
+  color: #67798e;
+}
+
+.img {
+  width: 100%;
+  height: auto;
+}
+
+.myProjects {
+  background: #fff;
+  width: 375px;
+  padding-bottom: 0;
+}
```

Comment:
- [ ] as part of the requirements please note that the my projects section should align items with `display: grid` kindly add it